### ISSUE TITLE
Always use srtime

### ISSUE THREAD

**Issue by e-carlin** [radiasoft/sirepo#2404]
Sometimes we use python's own datetime and other times we use srtime. I was working on a test and kept running up against my email auth token being expired. We [create the email token](https://github.com/radiasoft/sirepo/blob/36e1c7be155a69f175043e231bce13b55e171fec/sirepo/auth/email.py#L172) using datetime but [check it being expired](https://github.com/radiasoft/sirepo/blob/36e1c7be155a69f175043e231bce13b55e171fec/sirepo/auth/email.py#L68) using srtime. I've fixed this [instance in my work](https://github.com/radiasoft/sirepo/pull/2394/commits/93b96a329d473b1e889fff0b1765b75bb49c31bf) but I see uses of datetime elsewhere in the code.

On a separate but related note srtime "leaks" across functions in a test module. I'm not sure what the correct behavior is but it is something I didn't anticipate happening.

**Comment by robnagler** [comment]
Yes, srtime is a global. We could make it a bit more localized. You have to reset it at the end of the test.
